Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -593,13 +593,15 @@ public boolean onMenuItemClick(MenuItem item) {
                 menu.findItem(R.id.menu_checklist).setVisible(true);
                 menu.findItem(R.id.menu_history).setVisible(true);
                 menu.findItem(R.id.menu_markdown_preview).setVisible(mCurrentNote.isMarkdownEnabled());
+                menu.findItem(R.id.menu_sidebar).setVisible(true);
                 trashItem.setVisible(true);
             } else {
                 menu.findItem(R.id.menu_share).setVisible(false);
                 menu.findItem(R.id.menu_view_info).setVisible(false);
                 menu.findItem(R.id.menu_checklist).setVisible(false);
                 menu.findItem(R.id.menu_history).setVisible(false);
                 menu.findItem(R.id.menu_markdown_preview).setVisible(false);
+                menu.findItem(R.id.menu_sidebar).setVisible(false);
```

Comment:
This was one of the `else` branches to add the `menu.findItem(R.id.menu_sidebar).setVisible(false);` statement, but the [`else` branch below](https://github.com/Automattic/simplenote-android/pull/707/files#diff-6f25240fa3acbd8a012570ad7711f836R608) is still missing it.

Reply:
Done.